src/redux.js

```diff
@@ -696,15 +658,6 @@ export function getOptionFrequencies(state, column) {
   return optionFrequencies;
 }
 
-export function getOptionFrequenciesByColumn(state) {
-  let optionFrequenciesByColumn = {};
-  getSelectedCategoricalColumns(state).map(
-    column =>
-      (optionFrequenciesByColumn[column] = getOptionFrequencies(state, column))
-  );
-  return optionFrequenciesByColumn;
-}
-
```

Comment:
This can be deleted because we now show the bar charts instead of these numbers?

Reply:
I think so.  I discovered this uncalled code through Chrome's coverage feature, and then couldn't find any remaining calls to this function.

Reply:
It turns out that `getOptionFrequencies`, immediately above, takes a `column`, so it seems to serve the right purpose for `ColumnInspector`.